flask_resty/pagination.py

```diff
@@ -268,20 +280,83 @@ class CursorPaginationBase(LimitPagination):
 
     #: The name of the query parameter to inspect for the cursor value.
     cursor_arg = "cursor"
+    limit_arg = "limit"
 
     #: the name of the query parameter to inspect for explicit forward pagination
     after_arg = "after"
+    first_arg = "first"
 
     #: the name of the query parameter to inspect for explicit backward pagination
     before_arg = "before"
+    last_arg = "last"
 
     def __init__(self, *args, validate_values=True, **kwargs):
         super().__init__(*args, **kwargs)
         self._validate_values = validate_values
 
+    def try_get_arg(self, *args):
+        for arg in args:
+            value = flask.request.args.get(arg)
+            if value is not None:
+                return (value, arg)
+
+        return (None, None)
+
+    # There are a number of different cases that this covers in order to be backwards compatible with
+    def get_cursor_info(self):
```

Comment:
nitpick (non-blocking): might as well start doing this

```suggestion
    def get_cursor_info(self) -> CursorInfo:
```